src/components/LessonCard/LessonCard.tsx

```diff
@@ -108,63 +76,192 @@ const Paragraph = styled(P)`
+        </svg>
+      );
+  }
+}
+
+type MascotType = 1|2|3|4|5;
+type Progress = 0|1|2|3;
+
+type UnderlyingCardProps = {
+  isResourcesCard: boolean;
   lessonId: string;
+  mascotType: MascotType;
+  progress: Progress;
+  subtitle?: string;
+  title: string;
+  description: string;
+};
+
+type LessonCardProps = {
+  lessonId: string;
+  mascotType: MascotType;
+  progress: Progress;
   subtitle?: string;
   title: string;
   description: string;
-  call2actionText?: string;
+};
+
+type ResourcesCardProps = {
+  lessonId: string;
   mascotType: MascotType;
+  subtitle?: string;
+  title: string;
+  description: string;
 };
 
-export const LessonCard: FC<Props> = ({
+const UnderlyingCard: FC<UnderlyingCardProps> = ({
+  isResourcesCard,
   lessonId,
+  mascotType,
+  progress,
   subtitle,
   title,
   description,
-  call2actionText,
-  mascotType,
 }) => {
-  let mascotUrl;
-  switch (mascotType) {
-    case "lesson-1":
-      mascotUrl = shapesMascotLesson1Url;
-      break;
-    case "lesson-2":
-      mascotUrl = shapesMascotLesson2Url;
-      break;
-    case "lesson-3":
-      mascotUrl = shapesMascotLesson3Url;
-      break;
-    case "lesson-4":
-      mascotUrl = shapesMascotLesson4Url;
-      break;
-    case "lesson-5":
-      mascotUrl = shapesMascotLesson5Url;
-      break;
-    default:
-      mascotUrl = shapesMascotLesson1Url;
-      break;
-  }
   return (
     <Wrap>
+
       <BlockMascot $mascotType={mascotType}>
-        <ImageMascot src={mascotUrl} />
+        <DivMascot>
+          {selectMascot(mascotType, isResourcesCard, progress)}
```

Comment:
Related to my reply to the comment at https://github.com/FRUK-Simulator/Simulator/pull/292/files#r1579807948, this was actually intentional, so that the business logic of what assets make up "mascot" do not have to be exposed to the places that use the LessonCard. In terms of the asset lookup being done inside the component implementation if done the way as is, I think that can be mitigated by my proposal for just introducing an "AssetManager" component per my reply in there.